Update Django User Group page

maiksprenger opened the pull request:
Hello!

I am organizing the upcoming Django User Group events, and would like to update the description here. It mostly revolves around the Meetup group these days, so I've updated things accordingly. 

Thank you for review and maintenance of this super initiative!

maiksprenger committed:
Fix typo in README instructions

maiksprenger committed:
Update DJUB details

veit commented:
Hi Maik,

Thank you for updating the djub page. I would be happy if you could add the article-info directive again – then the look & feel of the pages will be more uniform. And you are also welcome to register as an organiser.

maiksprenger committed:
Re-add article-info

As requested in PR review.

maiksprenger commented:
Hi Vait, thanks for the prompt feedback. I added it again, please let me know if that looks alright. Since I rewrote most of the page, I thought it's best to put me as author?
